Soccar [Game-Server]/src/nl/soccar/gameserver/model/GameServer.java

```diff
@@ -25,22 +27,48 @@
     private GameServer() {
     }
 
+    /**
+     * Gets the Instance of the GameServer class.
+     * 
+     * @return GameServer The GameServer class.
+     */
     public static GameServer getInstance() {
-        return instance;
+        return INSTANCE;
     }
 
+    /**
+     * Registers the player.
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Document this public method. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://194.135.91.67:9000/coding_rules#rule_key=squid%3AUndocumentedApi)